docs/index.md

```diff
@@ -0,0 +1,35 @@
+# Testcontainers
+
+![Testcontainers Banner](banner.png)
+
+## About
+
+Testcontainers for .NET is a library to support tests with throwaway instances of Docker containers for all compatible .NET Standard versions. The library is built on top of the .NET Docker remote API and provides a lightweight implementation to support your test environment in all circumstances.
+
+Choose from existing pre-configured configurations and start containers within a second, to support and run your tests. Or create your own container images with Dockerfiles and run your containers and tests immediately afterward.
+
+## Supported operating systems
```

Comment:
We should add a section on Docker requirements, using this text (as in Java and Go):
https://www.testcontainers.org/supported_docker_environment/

Reply:
I've added the first part of System Requirements from testcontainers.org. We have to think about the details later. I think @mdelapenya brought up a good point regarding the compatible versions etc.